Cargo.toml

```diff
@@ -34,6 +35,8 @@ js-sys = { version = "^0.3" }
 libipld = { version = "0.16" }
 libipld-core = { version = "0.16" }
 libipld-cbor = { version = "0.16" }
+libipld-json = { version = "0.16" }
+multihash = { version = "0.18" }
```

Comment:
Note/no action required: we use `multihash`v17 from `libp2p` in `noosphere_ns` (and already have both versions in tree)